libs/filamat/src/MaterialVariants.cpp

```diff
@@ -42,4 +44,13 @@ std::vector<Variant> determineVariants(uint8_t variantFilter, bool isLit,
     return variants;
 }
 
+std::vector<Variant> determinePostProcessVariants() {
+    std::vector<Variant> variants;
+    for (size_t k = 0; k < filament::POST_PROCESS_VARIANT_COUNT; k++) {
+        variants.emplace_back(k, filament::backend::ShaderType::VERTEX);
```

Comment:
Maybe leave a TODO for future support of the variant filter (for instance to remove transparent variants if you know you don't need alpha).

Reply:
Good idea.